repldex/backend/website.py

```diff
@@ -520,6 +520,8 @@ async def middleware(request, handler):
 	request.is_admin = is_admin
 	request.discord_id = discord_id
 	resp = await handler(request)
+	if resp.status in [404, 500, 418]:
```

Comment:
i am legally obligated to request changes, so this should be a set (curly instead of square brackets) to be nanoseconds faster

Reply:
I am legally obligated to curse you and your descendants whenever you request tiny changes unrelated to program functionality

Reply:
also, done